src/components/collaborative-editing/diff/createOperationByStringsDiff.ts

```diff
@@ -0,0 +1,100 @@
+import { InsertOperation, OperationType, RemoveOperation } from './types';
+
+/**
+ * Function finds difference between two different strings
+ *
+ * @param str1 - original string
+ * @param str2 - new string that comes from the first
+ *
+ * @returns {object[]} array of operations
```

Comment:
if it could return several operations, so name the function `createOperationsByStringsDiff`. And the file as well.

Reply:
Done